### PR TITLE
Fixed Issue in gilab link tooltip

### PR DESCRIPTION
**Pull request by raghavaggarwal2308**
Heading and description text overlays in tooltip when crosses a certain limit
Before-
Screenshot from 2022-07-27 14-48-00

Now-
Screenshot from 2022-07-27 14-46-16

**Ticket-**
https://brightscout.atlassian.net/browse/MI-1919?atlOrigin=eyJpIjoiYjU4ZmE3NDY2MmMwNGJkZmFlMGFkYTZhMDAyODNkZTgiLCJwIjoiaiJ9

### CODE REVIEW

File: webapp/src/components/link_tooltip/tooltip.css [hunk 1]
```diff
@@ -73,6 +73,7 @@
     line-height: 1.25;
     overflow: hidden;
     font-size: 12px;
+    word-break: break-word;
```
Comment (manojmalik20) [inline webapp/src/components/link_tooltip/tooltip.css:76]: Add screenshots in the PR for how the UI looked before your fix and how it looks now.

File: README.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ The GitLab/Mattermost plugin documentation is currently being updated and reloca
 [![Release](https://img.shields.io/github/v/release/mattermost/mattermost-plugin-gitlab)](https://github.com/mattermost/mattermost-plugin-gitlab/releases/latest)
 [![HW](https://img.shields.io/github/issues/mattermost/mattermost-plugin-gitlab/Up%20For%20Grabs?color=dark%20green&label=Help%20Wanted)](https://github.com/mattermost/mattermost-plugin-gitlab/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22Up+For+Grabs%22+label%3A%22Help+Wanted%22)
 
-**Maintainer:** [@iomodo](https://github.com/iomodo)
+**Maintainer:** [@mickmister](https://github.com/mickmister)
```
[the remark this answers is not in the document]
Comment (raghavaggarwal2308) [reply README.md:12]: This change was not on my local.

[the remark this answers is not in the document]
Comment (manojmalik20) [reply README.md:12]: Take a pull from `master` in `link_tooltip` branch and push it. Then, take a pull from `link_tooltip` in branch `MI-1919` and push that.

**Comment by manojmalik20** [comment]
Looking at the screenshots, I think we were supposed to truncate the text with ellipses instead of wrapping the sentences. Can you confirm what is the behaviour in Gitlab?

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (raghavaggarwal2308) [reply README.md]: Done

**Review by manojmalik20 — APPROVED** [review]

**Pull request merged**